packages/hub/routes/card-spaces.ts

```diff
@@ -170,6 +170,21 @@ export default class CardSpacesRoute {
     ctx.type = 'application/vnd.api+json';
   }
 
+  async postProfileCategoryValidation(ctx: Koa.Context) {
+    if (!ensureLoggedIn(ctx)) {
+      return;
+    }
+
+    let profileCategory: string = ctx.request.body.data.attributes['profile-category'];
+    let errors = await this.cardSpaceValidator.validate({ profileCategory } as CardSpace);
```

Comment:
Do we need to add the profanity filter to profile category validation?

Reply:
probably… anywhere user-editable strings show up, really. I’m having build problems but will try this next week, thanks.

Reply:
That’s done now, thanks.